core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/security/elytron-tool/main/module.xml

```diff
@@ -40,7 +40,7 @@
         <module name="org.jboss.logging" />
         <module name="org.jboss.logmanager" />
         <module name="org.slf4j" />
-        <module name="org.wildfly.security.elytron-private" />
+        <module name="org.wildfly.security.elytron-private" services="import"/>
```

Comment:
I am not sure if this dependency should be `org.wildfly.security.elytron-base`, for the reference, the new module [org.wildfly.security.http.sfbasic](https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/security/http/sfbasic/main/module.xml#L32) has the elytron-base dependency.

Reply:
This could be changed to `org.wildfly.security.elytron-base` (`elytron-private` depends on `elytron-base` and a few other deps but it doesn't look like those are actually needed by `elytron-tool`).